### PR TITLE
Added java unit test and formatter stuff to the pom

### PR DESCRIPTION
**Pull request by jango2106**

### CODE REVIEW

File: server/pom.xml [hunk 1]
```diff
@@ -1,153 +1,185 @@
...
+      <organization>ShowMeTheRoapes</organization>
+      <roles>
+        <role>Software Developer</role>
+      </roles>
+    </developer>
+  </developers>
+  <issueManagement>
+    <system>Github Issues</system>
+    <url>https://github.com/ShowMeTheRoapes/Paracord/issues</url>
+  </issueManagement>
```
Comment (ahoopes16) [inline server/pom.xml]: Holy cow did you add some of this manually? 😲

Comment (jango2106) [reply server/pom.xml]: It kind of hates xml files, but yes I have added everything on this page :)

File: yarn.lock [hunk 2]
```diff
@@ -123,7 +123,7 @@ hosted-git-info@^2.1.4:
   resolved "https://registry.yarnpkg.com/hosted-git-info/-/hosted-git-info-2.7.1.tgz#97f236977bd6e125408930ff6de3eec6281ec047"
   integrity sha512-7T/BxH19zbcCTa8XkMlbK5lTo1WtgkFi3GvdWEyNuc4Vex7/9Dqbnpsf4JMydcfj9HCg4zUWFTL3Za6lapg5/w==
 
-husky@^2.4.1:
+husky@~2.4.1:
```
[the remark this answers is not in the document]
Comment (jango2106) [reply yarn.lock]: It did it

File: server/src/main/java/com/showmetheroapes/example/ExampleApplication.java [hunk 3]
```diff
@@ -2,28 +2,24 @@
...
-		System.out.println("Rest application has started. Hit enter to stop.");
-		System.in.read();
-		server.shutdown();
-	}
+  public static void main(final String[] args) throws IOException {
+    final HttpServer server = startServer();
+    System.out.println("Rest application has started. Hit enter to stop.");
+    System.in.read();
+    server.shutdown();
+  }
```
[the remark this answers is not in the document]
Comment (jango2106) [reply server/src/main/java/com/showmetheroapes/example/ExampleApplication.java]: Yes! It re-formats it during the "mvn fmt:format" step to follow Google's preferred style

File: package.json [hunk 4]
```diff
@@ -2,8 +2,8 @@
   "dependencies": {},
   "husky": {
     "hooks": {
-      "pre-commit": "cd paracord-ui && echo Linting UI files... && yarn lint-fix && cd ../",
-      "pre-push": "cd paracord-ui && echo Running UI tests... && yarn test && cd ../"
+      "pre-commit": "cd paracord-ui && echo Linting UI files... && yarn && yarn lint-fix && cd ../ && cd server && echo Formatting server files... && mvn fmt:format && cd../",
+      "pre-push": "cd paracord-ui && echo Running UI tests... && yarn && yarn test && cd ../ && cd server && echo Building server project... && mvn clean install && cd../"
```
[the remark this answers is not in the document]
Comment (jango2106) [reply package.json]: Fixed!

File: server/pom.xml (hunk 1 above)
Comment (ahoopes16) [reply server/pom.xml]: Ewwww lol but impressive sir :D

**Pull request merged**